lang/fpc/Portfile

```diff
@@ -14,8 +14,8 @@ description         Free Pascal, an open source Pascal and Object Pascal compile
 long_description    Free Pascal is a 32, 64 and 16 bit professional Pascal compiler. \
```

Comment:
This is still longer than 80 columns. I would put it in a favorite text editor and look at where column 80 is.

Reply:
Just to be clear: everything but the checksums and the mode line at top should probably be under 80 columns.

Reply:
@pmetzger Is it documented somewhere? I can many ports using much longer lines, and it makes it less readable and more ugly to force the text into such narrow field. Of course, if it is a declared requirement, no further argument on it. But then I wonder why it is not followed so often.

Reply:
This is our standard, yes. Many project members open up Portfiles in editors with 80 column lines. If some Portfiles you have seen routinely use wider lines, they are in error. I will note this is not a question of editor limitations, this is a question of readability.